chapter-2.md

```diff
@@ -175,8 +175,12 @@ test "make dir" {
     _ = try dir.createFile("y", .{});
     _ = try dir.createFile("z", .{});
 
+    const iterDir = try std.fs.cwd().openIterableDir(
```

Comment:
The dir doesn't need to be opened twice https://github.com/Sobeston/ziglearn/pull/154